USDFI & STABLE

tbotteam opened the pull request.

tbotteam committed:
USDFI & STABLE

RanGojo commented:
Hey @tbotteam, Thanks for the PR. 

- [ ] It it possible to update your changes (or create a new pr) and revert your changes except those mentioned in [readme](https://github.com/rango-exchange/token-list?tab=readme-ov-file#how-to-list-your-token-in-rango). (You need only to add your icons/manifests in tokens/custom directory. All other files like dest/rango-custom-tokens.json are auto generated/updated and there is no need to update them in PR. (Don't worry about svg icons, we will handle it in a later commit)

- [ ] Also as API team told me, your tokens names/symbols are not matched with names/symbols in your contract. So you need to use exact names here.

- [ ] Please also update PR description based on readme guide.

tbotteam committed:
Revert "USDFI & STABLE"

This reverts commit 479940043c5e53192b285ad8f45ea8152dcf69f3.

tbotteam committed:
USDFI & STABLE #2

BaseFactory: https://bscscan.com/address/0xB3863573d9f25e6a84895d4685a408db7a488416

supported by;
1inch,
openOcean,
Paraswap,
Odos

tbotteam commented:
BaseFactory: https://bscscan.com/address/0xB3863573d9f25e6a84895d4685a408db7a488416

supported by;
1inch,
openOcean,
Paraswap,
Odos,
Kyberswap
